core/cmd/local_client.go

```diff
@@ -155,7 +155,11 @@ func initLocalSubCmds(client *Client, devMode bool) []cli.Command {
 			Usage:       "Commands for managing the database.",
 			Description: "Potentially destructive commands for managing the database.",
 			Before: func(ctx *clipkg.Context) error {
-				return client.Config.ValidateDB()
+				err := client.Config.ValidateDB()
+				if err != nil {
+					prettyPrintConfigErr(err)
+				}
+				return err
```

Comment:
Should we return a different error too?
```suggestion
				return cli.errorOut(errors.New("invalid configuration"))
```

Reply:
It looks like this would dedupe the stdout/stderr messages.